cache-config/tm-health-client/config/config.go

```diff
@@ -196,8 +196,13 @@ func GetConfig() (Cfg, error, bool) {
 		return Cfg{}, errors.New(err.Error() + "\n"), false
 	}
 
-	if err = ReadCredentials(&cfg); err != nil {
-		return cfg, err, false
+	if cfg.TOPass == "" || cfg.TOUser == "" || cfg.TOUrl == "" {
+		if cfg.TOCredentialFile == "" {
+			return Cfg{}, errors.New("cannot continue, no TO credentials have been specified."), false
```

Comment:
error strings shouldn't end with punctuation.

Reply:
made the change